### PR TITLE
Add Anaconda icon

### PR DESCRIPTION
**Pull request by mvedataydin**
**Issue:**
#1719 

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I manually converted official .eps file to .svg.
The brand color is given in the related issue and from inspecting the icon.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3521,4 +3526,4 @@
             "source": "https://github.com/zulip/zulip/"
         }
     ]
-}
+}
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: Small request, but could you revert this change @mvedataydin? It's not a big deal, especially not in this project, but I think it is good practice not to pollute `git blame` and `git diff`. Especially in bigger/more complex projects and/or larger Pull Requests :slightly_smiling_face: 

You might have to do something with the configuration of your editor/IDE. Alternatively you can use a simple text editor (like notepad.exe or gedit) or even the GitHub online IDE to fix this. Let me know if you can't figure this out.

Comment (mvedataydin) [reply _data/simple-icons.json]: Thank you a lot for the feedback,I am slightly new to contributing to open source. I was trying to figure out and  solve the issue and mistakenly closed the pr. Do you have any suggestions to me with best practices of contributing? Thank you in advance.

**Comment by mvedataydin** [comment]
Thank you a lot for the feedback,I am slightly new to contributing to open source. I was trying to figure out and solve the issue and mistakenly closed the pr. Do you have any suggestions to me with best practices of contributing? Thank you in advance.

**Comment by ericcornelissen** [comment]
> I am slightly new to contributing to open source. I was trying to figure out and solve the issue and mistakenly closed the pr...

No problem, if you close a PR there should be a button on this page that allows you to reopen it (as long as the branch you worked on still exists). If you couldn't find it, that's fine for now and we'll just continue with #1728.


> ... closed the pr with force pushing a commit.

I'm not sure how that happened, as far as I know that shouldn't happen (unless you pushed with `--delete`). Anyway, there is no need (at least in this project) to force push your changes. We always [squash merge](https://help.github.com/en/articles/about-pull-request-merges) Pull Requests like this.


> Do you have any suggestions to me with best practices of contributing?

In general it differs from project to project. As for this Pull Request, everything looked great. So I cannot really give you any concrete advice now, but if you have any questions feel free to ask us here :slightly_smiling_face: 

Again, the additional newline at the end of the JSON file is not really a problem, though in larger Pull Requests things like that may become annoying and it is best if you try to avoid them/fix them before you open the Pull Request.

**Pull request closed without merging**